src/Map/src/Bridge/Google/src/Renderer/GoogleRenderer.php

```diff
@@ -37,6 +37,7 @@ public function __construct(
         private ?int $retries = null,
         private ?string $url = null,
         private ?string $version = null,
+        private array $libraries = [],
```

Comment:
Some type-hints maybe ? Also, not a big fan of non-nullable after nullable (but that's maybe just me)